packages/angular/projects/ui-angular/src/lib/components/authenticator/components/amplify-confirm-sign-in/amplify-confirm-sign-in.component.html

```diff
@@ -1,13 +1,4 @@
-<!-- 
-  Define default contents here
--->
-<ng-template #confirmSignInButton>
-  <button amplify-button variation="primary" fullWidth="true" type="submit">
-    {{ confirmText }}
-  </button>
-</ng-template>
```

Comment:
As we're closing GA, I wanted to be more careful about which slots we expose. As discussed, I'll incrementally open slots as we resolve customer use-cases. Currently, these slots are

1. `sign-up-form-fields` (in upcoming PR!)
2. top level`sign-in`, `sign-up`, ...

but we will be adding more slots like headers and footers as we go on.

Reply:
Yeah, I was thinking about this in Vue, if I should remove all the slots I have, and only include a few. I think both approaches work. This is fine